src/pages/applications/Applications1.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { Pagination, Skeleton } from "antd";
+import axios from "axios";
+
+import TableView from "../../components/TableView";
+import CommonFilters from "../../components/CommonFilters";
+
+import { useAuth } from "../../context/AuthContext";
+
+import useForm from "../../hooks/useForm";
+import useFilter from "../../hooks/useFilter";
+
+import { API_END_POINT } from "../../../config";
+
+
+const Applications = () => {
+  const filter = useFilter("applicant");
+
+  const { id: batchId } = useParams();
+  const { token } = useAuth();
+  const [applications, setApplications] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  const columnNameList = [
+    { key: "full name", title: "Full Name" },
+    { key: "dob", title: "DOB" },
+    { key: "email", title: "Email" },
+    { key: "district", title: "District" },
+    { key: "invite", title: "Invite" },
+    { key: "viewMore", title: "View More" },
+  ];
+
+  const { formData, handleChange } = useForm({
```

Comment:
Why are we using our custom hook, when we already discussed to handle all the input fields thorugh react-hook-form

Reply:
This file is not used in our application